### DISCUSSION TITLE
Support for the Discord website(?).

### DISCUSSION THREAD

**Discussion by CalinZBaenen** [Tyrrrz/DiscordChatExporter#1217] [Q&A]
Will this software work with the Discord website? If not, are there plans for a browser extension?

Personally, I prefer the website, so that I don't need more apps than necessary.  
It would be sweet to (know that I can) use this software with the Discord website.

My assumption, based on my programming background, is no, since this is a standalone application.

**Comment by Tyrrrz** [comment]
It doesn't use neither the website nor the app, it just needs your token. The application, however, is a desktop application, because a web application would require proxying Discord API calls through a separate backend (high traffic, privacy concerns) or working around CORS (same issue and/or impossible).
